tests/test.cpp

```diff
@@ -2841,6 +2843,40 @@ void FixedLengthArrayTest() {
 #endif
 }
 
+void NativeTypeTest() {
+// looks like VS10 does not support std::vector with explicit alignment
+// From stackoverflow (https://stackoverflow.com/questions/8456236/how-is-a-vectors-data-aligned):
+//
+// Visual C++ version 2010 will not work with an std::vector with classes whose alignment are specified.
+// The reason is std::vector::resize.
+// When compiling, next error appears:
+// c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\include\vector(870): error C2719: 
+//  '_Val': formal parameter with __declspec(align('8')) won't be aligned [C:\projects\flatbuffers\flattests.vcxproj]
+
+#if !defined(_MSC_VER) || _MSC_VER >= 1700
+  const int N = 3;
+
+  Geometry::ApplicationDataT srcDataT;
```

Comment:
please use Google C++ Style Guide for variable names, i.e use snake case rather than camel case

Reply:
fixed